README.rst

```diff
@@ -615,6 +615,11 @@ Choose from the list of available fixers:
                         closing semicolon are
                         prohibited.
 
+* **native_function_casing** [contrib]
```

Comment:
no, actually replace them :D

I updated :  https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1659/files#r49096853

sorry for confusion